global.json

```diff
@@ -1,5 +1,5 @@
 {
   "msbuild-sdks": {
-    "MSBuild.Sdk.Extras": "2.1.2"
+    "MSBuild.Sdk.Extras": "3.0.23"
```

Comment:
I don't mind the change, just curious - why was it necessary? Since we previously targeted .NET 5, it's a bit surprising that the package didn't support 3.1.

Reply:
There is a problem with importing `Microsoft.WinFX.props` on Linux. Reading [here](https://github.com/dotnet/sdk/issues/11108) there seem to be problems with MSBuildExtras in linux since April 2020 exactly with this `Microsoft.WinFX.props`. Even when the patch was in the sdk it didn't seem to work unless you "patch yourself" some settings. Before doing anything I just gave a shot to the latest version of MSBuild.Sdk.Extras just to see if that would have helped. Which it didn't, so I'll revert.

Reply:
I see. I guess we can use the workaround from that post then.